src/views/GameListView.jsx

```diff
@@ -145,15 +149,22 @@ const GameListView = () => {
       <Content>
         <Filter>
           <h2 className="incentive">{t("Start a game now")}</h2>
-          <input
-            type="search"
-            id="game-search"
-            name="game-search"
-            aria-label="Search for a game"
-            placeholder="Search for a game"
-            value={searchTerm}
-            onChange={handleChange}
-          />
+          <StyledGameFilters>
```

Comment:
Is there a more elegant way to do that with styled components? Or you just add styled components for each layer?

Reply:
Yes: you just need to define a styled wrapper then you can define style for children dom like in https://github.com/jrmi/airboardgame/blob/24ed9b5f18719ea873cce97047b5012381c53bfe/src/views/GameListItem.jsx#L9